digits/model/views.py

```diff
@@ -225,6 +233,51 @@ def download(job_id, extension):
     response.headers['Content-Disposition'] = 'attachment; filename=%s_epoch_%s.%s' % (job.id(), epoch, extension)
     return response
 
+@blueprint.route('/<job_id>/delete',
+        methods=['POST'])
+def delete(job_id):
```

Comment:
This route name is misleading. Can you format the route like `/<job_id>/delete_snapshot[.json]?epoch=<epoch>` instead?